dev/sbt-plugin/src/main/scala-sbt-1.0/com/lightbend/lagom/sbt/DynamicProjectAdder.scala

```diff
@@ -120,7 +121,8 @@ package com.lightbend.lagom.sbt {
 
         // Now we recreate the structure, this is where the structure data is calculated, which evaluates all the settings
         // and works out all the dependencies.
-        val reindexedStructure = sbt.LagomLoad.reapply(newSession.mergeSettings, structureWithNewProject)
+        val logger = extracted.get(Keys.sLog in ThisBuild)
```

Comment:
@renatocaval, I think that using `sLog` is better here. `sLog` is a well-known public API in sbt, so it is probably safer.